aptos-move/framework/aptos-framework/sources/Account.move

```diff
@@ -343,18 +343,70 @@ module AptosFramework::Account {
     }
 
     ///////////////////////////////////////////////////////////////////////////
-    /// Basic account creation method.
+    /// Basic account creation methods.
     ///////////////////////////////////////////////////////////////////////////
 
     public(script) fun create_account(auth_key: address) {
-        let (signer, _) = create_account_internal(auth_key);
+        let signer = create_account_internal(auth_key);
         Coin::register<TestCoin>(&signer);
     }
 
+    public fun create_module_account(
```

Comment:
this is a weird function without doc, I was expecting each account owner can extract a signer capability itself and rotate private key to 0x0 or something

Reply:
No, let's not do that. Let's make this a part of a complex app registration. Might be a good chance for @areshand initialize function ;).